main.yml

```diff
@@ -604,6 +605,22 @@ rhel-sw:
       - id: x86
         title: x86
 
+ros:
+  title: Resource Optimization
+  api:
+    versions:
```

Comment:
@kruai, as of now there is no API version released so will it be possible to add changes without this `versions` value? or is it mandatory to have?

Reply:
Looks like Karel answered this for you below, so I'm resolving :)